test/suites/one-node/test_tanssi_one_node.ts

```diff
@@ -190,11 +190,18 @@ async function registerEmptyParathread(api, manager, paraId) {
             tx1
         )
     );
-    const bootNodes = ["/ip4/127.0.0.1/tcp/33051/ws/p2p/12D3KooWSDsmAa7iFbHdQW4X8B2KbeRYPDLarK6EbevUSYfGkeQw"];
-    const tx2 = api.tx.dataPreservers.setBootNodes(paraId, bootNodes);
-    txs.push(tx2);
-    const tx3 = api.tx.registrar.markValidForCollating(paraId);
-    txs.push(tx3);
+
+    const profileId = await api.query.dataPreservers.nextProfileId();
+    txs.push(
+        api.tx.dataPreservers.createProfile({
```

Comment:
In all zombienet tests we should use `forceCreateProfile`, copy it from test/suites/parathreads/test_tanssi_parathreads.ts

This currently fails if you run

pnpm moonwall test zombie_tanssi_one_node

Reply:
it depends how we do it though, but in general yes, any batch call executed with sudo should do that

Reply:
It was only this test that was failing, I fixed it and pushed a commit